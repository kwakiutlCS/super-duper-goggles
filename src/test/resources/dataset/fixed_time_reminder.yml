reminder:
- id: 1
  content: content
  userId: user
  createdAt: 1000
  updatedAt: 1000
  time_id: 2
time:
- id: 2
  time: 3600 