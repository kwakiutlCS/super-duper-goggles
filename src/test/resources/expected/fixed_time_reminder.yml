reminder:
- id: 1
  content: content
  userId: user
  createdAt: 1000
  updatedAt: 1000
time:
- id: 2
  time: 7200